src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/__mocks__/formDataStateMock.ts

```diff
@@ -0,0 +1,15 @@
+import { IFormDataState } from '../src/features/form/data/formDataReducer';
+
+export function getFormDataStateMock(customState?: Partial<IFormDataState>) {
+  const formData: IFormDataState = {
+    error: null,
+    formData: {},
+    hasSubmitted: false,
+    isSaving: false,
+    isSubmitting: false,
+    responseInstance: false,
+    unsavedChanges: false,
+  };
+
+  return { ...formData, ...customState };
+}
```

Comment:
GitHub doesn't seem to like these...
![image](https://user-images.githubusercontent.com/6088624/86586933-0a276480-bf89-11ea-8d24-f4ec55966dec.png)

Reply:
Yep, eslint doesn't like them either :) fixed.